### PR TITLE
Update Knative Serving and friends to 0.25.1

### PR DESCRIPTION
**Pull request by markusthoemmes**
As per title. To be merged after we cut the 1.18 branch. Just prepping things for now.

### CODE REVIEW

**Comment by markusthoemmes** [comment]
Depends on #1212

**Comment by markusthoemmes** [comment]
Uargh, will have to cut the respective eventing releases and publish their images first.

File: openshift-knative-operator/cmd/operator/kodata/ingress/0.25/0-networkpolicy-mesh.yaml [hunk 1]
```diff
@@ -16,15 +16,15 @@ spec:
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
-  name: istio-webhook
```
Comment (devguyio) [inline openshift-knative-operator/cmd/operator/kodata/ingress/0.25/0-networkpolicy-mesh.yaml:19]: @markusthoemmes I've no clue here so I'm guessing that's planned? I saw a patch removing the "net" for kourier stuff so just making sure

**Review by devguyio — COMMENTED** [review]
LGTM, just a quick check then I'll lgtm

File: openshift-knative-operator/cmd/operator/kodata/ingress/0.25/0-networkpolicy-mesh.yaml (hunk 1 above)
Comment (markusthoemmes) [reply openshift-knative-operator/cmd/operator/kodata/ingress/0.25/0-networkpolicy-mesh.yaml:19]: Yep this is expected and shouldn't impact anything.

**Pull request merged**